src/user/user.controller.ts

```diff
@@ -14,12 +14,18 @@ export class UserController {
   constructor(private readonly userService: UserService) {}
 
   @Post()
+  @ApiOperation({
+    description: 'Stores basic profile data for a user',
+  })
   @ApiBody({ type: CreateUserDto })
   async createUser(@Body() createUserDto: CreateUserDto): Promise<GetUserDto> {
     return await this.userService.createUser(createUserDto);
   }
 
-  @ApiBearerAuth()
+  @ApiBearerAuth('access-token')
+  @ApiOperation({
+    description: 'Returns basic profile data including partner access and course/session data',
```

Comment:
Returns user profile data with their nested partner access, partner admin, course user and session user data.